commerce-inventory-service/service.xml

```diff
@@ -188,7 +188,7 @@
 		<finder name="CommerceInventoryWarehouseId" return-type="Collection">
 			<finder-column name="commerceInventoryWarehouseId" />
 		</finder>
-		<finder name="C_S" return-type="CommerceInventoryWarehouseItem">
+		<finder name="C_S" return-type="CommerceInventoryWarehouseItem" unique="true">
```

Comment:
If we do want to make this index unique, I think we'll need to write an upgrade process to deduplicate this table, right? Or was it already impossible to create two rows with the same `commerceInventoryWarehouseId, sku` combination? I think even if it was already impossible, we should at least run a `DummyUpgradeProcess` to mark the change.

(But see my comment on the local service impl changes -- I think we may not want to make this finder unique.)

Reply:
Hi Ethan, may I ask you some clarification on this? Anyway AFAIK it should be already impossible to create two rows with the same combination.

Reply:
Hi @ethanbustad , thank you I really appreciate your explanation as it gave a deeper knowledge of how Liferay framework works, but I was wondering why you doubt we need a unique index on those columns, because after some talks we ended in consider it necessary

Reply:
Oh I see. Just because (1) it would/could require deduplicating the database first, and (2) if `sku` is ever blank, then the combination of `commerceInventoryWarehouseId+sku=blank` may not be unique (but if `sku` can't be blank then this isn't a problem).